sled-hardware/src/illumos/mod.rs

```diff
@@ -58,6 +59,9 @@ enum Error {
     #[error("Failed to issue request to sysconf: {0}")]
     SysconfError(#[from] sysconf::Error),
 
+    #[error("Node {node} missing device insance")]
```

Comment:
s/insance/instance/

Reply:
Fixed in [9cf9742](https://github.com/oxidecomputer/omicron/pull/6026/commits/9cf974262969bcd8d1047406be7a37bbd291ea49)